packages/recomponents/package.json

```diff
@@ -38,6 +39,7 @@
     "yarn": ">= 1.10.1"
   },
   "dependencies": {
+    "core-js": "^3.4.3",
```

Comment:
What is the reason this becomes a non-dev core dependency now?

Reply:
Upgradin `@vue/cli` also forced us to update a few plugins - one of them is `@vue/cli-plugin-babel` which now requires `core-js` as a peer dependency

https://cli.vuejs.org/migrating-from-v3/#vue-cli-plugin-babel-vue-babel-preset-app

Reply:
🤔 I used the automated command, which added it to `dependencies`
I'll test it out with the newer version of `core-js` still in dev-deps just to see if it still works. No reason to ship a new core dependency without a legitimate need for it

Reply:
Looks like it all works fine with `core-js` in dev deps, so I've moved it back